integration-testing/playwright.config.ts

```diff
@@ -0,0 +1,24 @@
+import * as path from "node:path";
+import { defineConfig, devices } from "@playwright/test";
+import * as dotenv from "dotenv";
+
+dotenv.config({ path: path.resolve(__dirname, ".env") });
```

Comment:
Is this needed despite not having a `.env` file? If so, the dotenv package can be removed too